python/nano/src/bigdl/nano/pytorch/inference/optimizer.py

```diff
@@ -52,28 +49,70 @@
 os.environ['LOGLEVEL'] = 'ERROR'  # remove parital output of inc
 
 
+class TorchAccelerationOption(AccelerationOption):
```

Comment:
I think this is better than the current code.

But what I meant is that each `AccelerationOption` has its own `optimize` method.

Say for `bf16_ipex`, we can has a `BF16IPEXAcceleration` class:

```python
class BF16IPEXAcceleration:

    def optimize():
        InferenceOptimizer.quantize(...)

class FP32JITAccelerator:

    def optimize():
        InferenceOptimizer.trace(...)
```

Reply:
I think this PR can be a first step to simplify.

Reply:
Thanks for suggestions! Will merge this PR first, and try above method in next PR.